process_environment.py

```diff
@@ -9,14 +9,14 @@
 
 DEBUG = False
 
-envIrradiance_cmd = "envIrradiance"
-envPrefilter_cmd = "envPrefilter"
-envIntegrateBRDF_cmd = "envBRDF"
-cubemap_packer_cmd = "cubemapPacker"
-panorama_packer_cmd = "panoramaPacker"
-envremap_cmd = "envremap"
-samplesGGX_cmd = "samplesGGX"
-envBackground_cmd = "envBackground"
+envIrradiance_cmd = "./envIrradiance"
```

Comment:
means you have "./" in the PATH. 
not the case on the VM.
I'll make myself a shell script encapsulator to launch the py then.